c++/src/capnp/serialize-packed.c++

```diff
@@ -439,6 +439,8 @@ PackedMessageReader::PackedMessageReader(
 
 PackedMessageReader::~PackedMessageReader() noexcept(false) {}
 
+#ifndef KJ_NO_FD
```

Comment:
Our preferred style throughout KJ code is `#if !KJ_NO_FD`, relying on the fact that undefined macros are implicitly falsy, rather than checking whether the macro is defined. (Note that the compiler flag `-D` defaults to using a value of `1`, so it works with this style.)